conformance-checkers/tools/picture.py

```diff
@@ -2,6 +2,7 @@
 import os
 ccdir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 template = """<!DOCTYPE html>
+<!-- DO NOT EDIT! This test has been generated by tools/picture.py. -->
```

Comment:
Full root-relative path is probably more helpful for locating the file

```suggestion
<!-- DO NOT EDIT! This test has been generated by /conformance-checkers/tools/picture.py. -->
```

Reply:
> Full root-relative path is probably more helpful for locating the file

Yep — made it so